ui/app/src/modules/Preview/Tools/ContentTree.tsx

```diff
@@ -308,19 +295,35 @@ function TreeItemCustom(props: TreeItemCustomInterface) {
     return () => isMounted.current = false;
   }, []);
 
+  if (!nodes) {
```

Comment:
When why would a node be null?
Does moving the code chunk far from its prior line numbers have a purpose? If the nullish node is in fact a valid circumstance, returning earlier seems more efficient.

Reply:
Yes, in this case I found it necessary because it seems that the component gets call with the nodeId of the selected item on child trees that are close and not render by default after the parent tree changes.